### ISSUE TITLE
ImGui::mouse_delta broke after update to imgui 1.65.2

### ISSUE THREAD

**Issue by malikolivier** [Gekkio/imgui-rs#166]
`ImGui::mouse_delta` now returns values extremely close to 0.

For example, after adding the following line to the `hello_world.rs` example,

```rust
let mouse_delta = ui.imgui().mouse_delta();
println!("Mouse Delta: {:?}", mouse_delta);
```

Only the following kind of lines will be printed on my machine running Ubuntu 18.04 with X11 (on commit b4edc01 and current latest master 8bf6831).

```
Mouse Delta: (0.000000000000000000000000000000000000000000589, 0.000000000000000000000000000000000000000001265)
```

We would instead expect mouse delta to print the number of pixels the mouse has moved between two frames. It works as expected on commit c85e73e.

I currently do not know why such regression happened, and I am going to investigate. I will update this issue when I have more information. @Gekkio Do you have any thought or idea?

**Comment by Gekkio** [comment]
There was a struct layout problem in `ImGuiIO` that I fixed just now with 1750aa5e72e630959f4b019c8bb7017c133e58bc

Could you check if that fixes the problem?

**Comment by malikolivier** [comment]
I went an a walk and when I was back it was fixed! Awesome!

I confirm that commit 1750aa5 fixes the error raised by this issue.
